config/localTemplate.go

```diff
@@ -452,6 +452,9 @@ type Local struct {
 	// MaxAcctLookback sets the maximum lookback range for account states,
 	// i.e. the ledger can answer account states questions for the range Latest-MaxAcctLookback...Latest
 	MaxAcctLookback uint64 `version[23]:"8"`
+
+	// EnableTransactionSimulator enables the simulation API endpoint.
+	EnableTransactionSimulator bool `version[23]:"true"`
```

Comment:
This should probably be false, since `EnableDeveloperAPI` is default false. We don't want relays, for example, to suddenly let people hammer them with these.

Reply:
Good point about relays. I suggesting defaulting to true since if this feature is used to implement read-only app calls and to help identify which foreign references an app call txn needs, it seems like it should be part of the standard algod setup.

But it may be too early to make a decision about this, in which case I'd be ok with defaulting to false and revisiting before releasing.

Reply:
I suppose the normal developer token might be enough safety - that's what prevents submitting txns directly to relays.

Reply:
At the current moment, I changed this to default to `false` since, as Jason said, we can always revisit this before releasing.